extensions/src/Wav2B64/Wave.h

```diff
@@ -6,53 +6,48 @@
 #include <string>
 
 
-typedef struct __WAVEDESCR
+typedef struct __waveDescr
```

Comment:
This is a library struct
https://github.com/xinshangbo/WendySpeech/blob/master/src/CutVoiceCPP/Wave.h#L6